site/content/entry/yuzu-progress-report-nov-2023/index.md

```diff
@@ -0,0 +1,359 @@
+NCE also saves up to 128MB of memory per guest CPU core (of which the Switch has four), meaning up to 512MB of RAM are saved by dropping the JIT.
+
+Having a device with active cooling will still produce the best results, but with NCE, performance is higher, battery life is higher, heat output is lower, and games load faster.
+The FPS boost is usually around 20-100%, depending on the game and device.
+
+{{< single-title-imgs
+    "Results aren’t comparable between the two devices, Mali was run at 0.5x to reduce its bottleneck"
+    "./nce1.png"
+    "./nce2.png"
+    >}}
+
+Sadly, as always, it’s not all perfect–"restrictions may apply."
+Not all games can take advantage of NCE. 
+Due to limitations in the Linux kernel and the requirements to run on a very restricted memory address space, some early Switch games need to have their updates installed to be able to run with NCE.
+Games like `The Legend of Zelda: Breath of the Wild` and `ARMS` use a deprecated 36-bit address space in their original v1.0.0 release, and NCE requires 39-bit to work. Here the emulator automatically reverts to JIT to allow booting the game.
+These games get full 39-bit support only with their respective updates installed.
+You can confirm this when enabling the framerate counter, which will show if NCE or JIT is in use:
+
+{{< single-title-imgs
+	"As simple as that (The Legend of Zelda: Breath of the Wild)"
+	"./jit.png"
+	"./nce.png"
+>}}
+
+Another limitation is 32-bit games. `Mario Kart 8 Deluxe` and `Captain Toad: Treasure Tracker`, among others, just won’t work with NCE, and the emulator will, just like with 36-bit games, automatically revert to using the [Dynarmic](https://github.com/merryhime/dynarmic) JIT.
+
+While there are ways to be able to use NCE with 32-bit games, our investigation found that it would require a secondary 32-bit build target, which we have never supported, and it is no longer viable on Android API versions for Android 11 or later, which happens to be our minimum requirement.
+
+There are also rare cases of games refusing to work with NCE, due to Linux being less permissive than the Switch operating system. One example is `Paper Mario: The Origami King`, which needs to have NCE manually disabled to run correctly for the moment.
+
+These limitations apply to both Android devices and other ARM devices like Linux ARM boards or laptops.
+
+We enable NCE by default on the latest GitHub and Play Store builds. The option can be found in `Settings > Advanced settings > Debug > CPU backend`.
+
+{{< imgs
+	"./cpu.png| Doesn’t hurt to double check"
+  >}}
+
+For the setting tweakers out there, disabling Fastmem disables NCE with it, so we don’t recommend disabling it unless you cannot run the game you want otherwise.
+If you see old pictures of games running with Fastmem disabled but still showing NCE under the framerate counter, that’s because the counter didn’t consider this scenario originally, it has since then been patched to correct this.
```

Comment:
```suggestion
If you see old pictures of games running with Fastmem disabled but still showing NCE under the framerate counter, that’s because the counter didn’t consider this scenario originally; it has since then been patched to correct this.
```